src/rules/no-large-snapshots.js

```diff
@@ -10,8 +10,30 @@ const reportOnViolation = (context, node) => {
   const startLine = node.loc.start.line;
   const endLine = node.loc.end.line;
   const lineCount = endLine - startLine;
+  const whitelistedSnapshots =
```

Comment:
we could verify all paths are absolute.

```js
const path = require('path');

const allPathsAreAbsolute = (whitelistedSnapshots || []).every(path.isAbsolute);

if (!allPathsAreAbsolute) {
  throw new Error('All paths must be absolute. You can use JS config and `path.resolve`');
}
```

That way it might be easier for users to use this rule correctly

Reply:
done